src/main/java/frc/robot/commands/InterfaceGetBranchLevelCommand.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.InterfaceSubsystem;
+import edu.wpi.first.wpilibj2.command.Command;
+
+/** An example command that uses an example subsystem. */
+public class InterfaceGetBranchLevelCommand extends Command {
```

Comment:
```suggestion
public class InterfaceBranchLevelCommand extends Command {
```

These _set_ levels right? But nonetheless you should name this in a more open way for flexibility.